filament/backend/src/vulkan/VulkanBinder.h

```diff
@@ -166,23 +166,16 @@ class VulkanBinder {
     // The pipeline key is a POD that represents all currently bound states that form the immutable
     // VkPipeline object. We apply a hash function to its contents only if has been mutated since
     // the previous call to getOrCreatePipeline.
-    struct alignas(8) PipelineKey {
+    #pragma pack(push, 1)
```

Comment:
If you used `__attribute__((packed))` you could make a define in `compiler.h`.
```
struct __attribute__((packed)) {
};
```

I'm not sure if this would work on Visual Studio though :-\

Reply:
Done, if it passes Kokoro then I'll ship it.